<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moneyphant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="TransferTab" name="transferTab">
       <attribute name="title">
        <string>Transfers</string>
       </attribute>
      </widget>
      <widget class="AccountTab" name="accountTab">
       <attribute name="title">
        <string>Accounts</string>
       </attribute>
      </widget>
      <widget class="BalanceTab" name="balanceTab">
       <attribute name="title">
        <string>Balance</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Statement_File"/>
    <addaction name="actionExport_Transfers"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Moneyphant"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTransfers"/>
    <addaction name="actionAccounts"/>
    <addaction name="actionBalance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionMerge_Selected_Accounts">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Merge Selected Accounts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionExport_Transfers">
   <property name="text">
    <string>Export Transfers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout_Moneyphant">
   <property name="text">
    <string>About Moneyphant</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionImport_Statement_File">
   <property name="text">
    <string>Import Statement File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionTransfer_View">
   <property name="text">
    <string>Transfer View</string>
   </property>
  </action>
  <action name="actionTransfers">
   <property name="text">
    <string>Transfers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionBalance">
   <property name="text">
    <string>Balance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TransferTab</class>
   <extends>QWidget</extends>
   <header>src/ui-tabs/TransferTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccountTab</class>
   <extends>QWidget</extends>
   <header>src/ui-tabs/AccountTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BalanceTab</class>
   <extends>QWidget</extends>
   <header>src/ui-tabs/BalanceTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
